rfcs/text/0000-simple-updates.md

```diff
@@ -0,0 +1,161 @@
+
+This RFC proposes a simple technical design to support the simplest metadata updates and additions, recognizing that this is the first step towards a complete solution.
+
+### Definitions
+
+***Addition*** - adding a metadata field:value pair to a metadata document where the metadata field existed in the schema version described by the metadata document  
+***Update*** - changing the value of an existing metadata field in a metadata document
+***Old update process*** - the current update process which involves adding the existing project to an exclusion list and re-ingesting the entire project - with updates - from scratch as a new submission  
+***Data wrangler*** - member of the DCP team who interacts with data contributors to collect metadata updates and additions and drives the process of submitting them to the DCP  
+***Experimental design representation*** - the connected set of entities which the DCP uses to represent the experimental design of a project  
+***Bundles*** - The data store representation of a logical set of data and metadata. There are two types of bundles input and analysis. Input bundles represent a single set of experimental data such as a lane or plex of sequencing and are defined by ingest as a biomaterial to file transition, connected by an assay process and all entities upwards in the experimental design from that point. Analysis bundles are all the analysis results from a single run of an analysis pipeline on a single input bundle and are defined by ingest as a file to file transition as defined by an analysis process and all entities upwards in the experimental design from that point.
+
+## User Stories
+
+As a data contributor, I would like to add the details of a scientific publication to my submission to the DCP to broaden knowledge of the work.
+
+As a data contributor, I would like to correct an error I have made in my submission to ensure that the correct metadata is presented to consumers of my data.
+
+As a data wrangler, I would like to update organ system metadata to reflect an improved ontology annotation for this particular organ.  
+
+Read more on the general user experience background for this work: [User Experience Problem Space - Top task “Update my project”](https://docs.google.com/document/d/1FP6kJa3f9NkOHTLqmkNYztls7soZLgeGefxdgR81ONE/edit)
+
+## Scientific "guardrails"
+
+The proposed design has consequences for end users where different types of updates will have different impact on how data is visible in the DCP. Updates and additions which are in scope for this solution will have provenance trails which could be displayed to users but other updates such as those which change the experimental design representation or require reanalysis will still use the old update process and won't have any provanance reflected inside the DCP. 
```

Comment:
Spelling. Second instance of *provanance* -> provenance - also, *auditable trail* is used below so you may wish to choose a common phrase. But foolish consistency and hobgoblins ... And that's a really long sentence that I'd break up for easier reading.

Reply:
Should be fixed and I made the audit trail consistent with provenance, happy for you to make further suggestions to make the text more readable